ideascube/monitoring/models.py

```diff
@@ -127,6 +127,10 @@ class Specimen(models.Model):
 
     objects = SpecimenQuerySet.as_manager()
 
+    @property
+    def user_identifier(self):
```

Comment:
I named it like that because this is the identifier from the user point of view.
But yes, maybe `pretty_indetifier` is better.